### ISSUE TITLE
Update the messaging app screenshot

### ISSUE THREAD

**Issue by MinThaMie** [ember-learn/guides-source#1898]
`/images/ember-core-concepts/messaging-app-1.png` is not showing the label that should be there according to accessibility. #1848 suggested to remove the label, but discussed with @MelSumner this should not be de solution.

<img width="719" alt="Screenshot 2023-02-07 at 14 51 55" src="https://user-images.githubusercontent.com/5811560/217263671-abee4bf8-b90b-4e9e-a9dc-6bc7cc3dc702.png">

<img width="429" alt="Screenshot 2023-02-07 at 14 52 27" src="https://user-images.githubusercontent.com/5811560/217263837-1ad49629-545d-459e-9c23-e9a6cb5b21f8.png">

**Comment by MinThaMie** [comment]
![Screenshot 2023-02-07 at 16 34 20](https://user-images.githubusercontent.com/5811560/217289982-9ad6bcd9-2307-4ebf-aab2-94784317a5a9.png)

If you copy-paste the current `hbs` and `css` this is the result
